app/commands/user/update.rb

```diff
@@ -0,0 +1,42 @@
+class User
```

Comment:
@iHiD Could you check if you are happy with this?

Reply:
@kntsoriano I refactored it to use Mandate Callbacks 🙂